src/build/app/mod.rs

```diff
@@ -431,7 +436,7 @@ impl<'help> App<'help> {
     /// // continued logic goes here, such as `app.get_matches()` etc.
     /// ```
     ///
-    /// [`App::from`]: ./struct.App.html#method.from
+    /// [`App::from_yaml`]: ./struct.App.html#method.from_yaml
```

Comment:
I don't think we have `from_yaml` anymore since it's been superseded by `impl From<&Yaml> for App`.

Reply:
Should also be fixed at the line 420.

Reply:
I couldn't figure out how to reliably link to the `From` impl, so in f6e28c5 I just removed the link.